src/java/apiview-java-processor/pom.xml

```diff
@@ -101,6 +127,41 @@
           </execution>
         </executions>
       </plugin>
+        <plugin>
```

Comment:
nit: Do we have some weird indentation here?

Reply:
Indeed, fixed that.